src/aap_eda/services/activation/engine/podman.py

```diff
@@ -351,6 +351,10 @@ def _pull_image(
     ) -> Image:
         try:
             log_handler.write(f"Pulling image {request.image_url}", True)
+            log_handler.write(
+                f"Log tracking id: {request.log_tracking_id}",
```

Comment:
Hi Hui, why you print here the log_tracking_id but not in the rest messages emitted by the engines?
Is it supposed that we are going to standardize the log_tracking_id for all the log entries in the manager/engine?

Reply:
@Alex-Izquierdo The purpose of this line is to display the ID on the UI detailed output page. All other log entries are standardized as described in the summary.